src/jsonValidator.js

```diff
@@ -0,0 +1,17 @@
+import { createError } from './error.js';
+
+export const jsonIsValid = (jsonString, errors) => {
+  try {
+    const checkJson = JSON.parse(jsonString);
```

Comment:
The way we're using this is that we only want to catch an error from the `JSON.parse(...)` function, no need to assign it to a const.